etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -104,6 +104,8 @@
 
 - code: "ASM"
   name: "American Samoa"
+  aliases:
```

Comment:
Running the harmonizer automatically adds new entries to `regions.yml`. This is beneficial, but it triggers an almost full ETL rebuild since many datasets depend on regions. Therefore, we try to avoid making any updates to it unless necessary (a paradox, right?).

In your case, it looks like you could remove the `*` suffix from country names, and you wouldn't need these aliases.

Reply:
Sounds good! I reverted the changes in this regions.yml file - is that sufficient for this? the countries.json is already created after all :D